hbase-server/src/test/java/org/apache/hadoop/hbase/HBaseTestingUtil.java

```diff
@@ -330,10 +329,13 @@ public HBaseTestingUtil(@Nullable Configuration conf) {
     }
     // Every cluster is a local cluster until we start DFS
     // Note that conf could be null, but this.conf will not be
-    String dataTestDir = getDataTestDir().toString();
-    this.conf.set("fs.defaultFS", "file:///");
-    this.conf.set(HConstants.HBASE_DIR, "file://" + dataTestDir);
-    LOG.debug("Setting {} to {}", HConstants.HBASE_DIR, dataTestDir);
+    if (this.conf.get("fs.defaultFS") != null && !this.conf.get("fs.defaultFS").startsWith("hdfs://")) {
```

Comment:
I guess here you do not want to override the fs.defaultFS config if it is present? But the logic here will skip setting fs.defaultFS if it is not set? Is this correct?